inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/questions/RequestAnswer.java

```diff
@@ -0,0 +1,88 @@
+package uk.ac.ucl.rits.inform.informdb.questions;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.time.Instant;
+
+/**
+ * Questions generated from specific data types (e.g. lab samples or consultation requests) trigger answers that or
+ * held as RequestAnswers. RequestAnswers therefore not only hold the answer as such but also link the answer to the
+ * question and the entity (i.e. data type) that triggered the question.
+ * @author Anika Cawthorn
+*/
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@AuditTable
+public class RequestAnswer extends TemporalCore<RequestAnswer, RequestAnswerAudit> {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long answerId;
```

Comment:
entity id should always be the entity name + Id

```suggestion
    private long requestAnswerId;
```